MonstieWash/Assets/Scripts/Animation Control/MonsterAnimControl.cs

```diff
@@ -0,0 +1,62 @@
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+[Serializable]
+public class MoodToAnimName
+{
+    public string moodName;
+    public string animName;
+}
+
+public class MonsterAnimControl : MonoBehaviour
+{
+    private MonsterBrain m_mimicAI;   
+    private string m_currentMood;   // tracks the monster's current mood (accurate to last frame)
+    private Animator m_myAnimator;
+
+    [SerializeField] private List<MoodToAnimName> moodToAnimationMap; // maps the name of moods to their animation names
```

Comment:
Since this has no methods or constructor, it makes more sense for it to be a struct. 
Also since it is only used in this class it makes more sense for it to be private and also inside the `MonsterAnimControl` class because (right now at least) other classes don't need to use it.
Finally, using strings for this sorta stuff is very volatile and error-prone so I've made some adjustments to make the lives of everyone easier. I've also removed `m_currentMood` in the below code since it wasn't being used for anything, but feel free to keep it if you've got plans for it.
```suggestion
public class MonsterAnimControl : MonoBehaviour
{
    [SerializeField] private List<MoodToAnimation> moodToAnimationMap = new(); // maps the name of moods to their animation names

    private MonsterBrain m_mimicAI;
    private Animator m_myAnimator;
    
    [Serializable]
private struct MoodToAnimation
{
    public MoodType mood;
    public AnimationClip animation;
}
```